Dockerfile

```diff
@@ -11,23 +23,28 @@ RUN source ~/.cargo/env && \
 
 COPY Cargo.toml .
 COPY Cargo.lock .
-ADD node /creditcoin-node/node
-ADD pallets /creditcoin-node/pallets
-ADD primitives /creditcoin-node/primitives
-ADD runtime /creditcoin-node/runtime
-ADD sha3pow /creditcoin-node/sha3pow
-ADD chainspecs /creditcoin-node/chainspecs
-ADD test /creditcoin-node/test
-RUN apt-get update && apt-get install -y protobuf-compiler
+COPY node /creditcoin-node/node
+COPY pallets /creditcoin-node/pallets
+COPY primitives /creditcoin-node/primitives
+COPY runtime /creditcoin-node/runtime
+COPY sha3pow /creditcoin-node/sha3pow
+COPY chainspecs /creditcoin-node/chainspecs
+COPY test /creditcoin-node/test
 RUN source ~/.cargo/env && cargo build --release
 
-FROM ubuntu:20.04
+FROM ubuntu:22.04
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends ca-certificates && \
+    apt-get upgrade -y && \
```

Comment:
We should probably run this upgrade before running the install

Reply:
I don't think the order actually matters but I've updated it anyway so that there is a consistency in the order of commands for the 2 images.